modules/ROOT/pages/start-using-sdk.adoc

```diff
@@ -1,26 +1,50 @@
 = Install and Start Using the Node.js SDK with Couchbase Server
 :navtitle: Start Using the SDK
 
-== Installing the Server
+[#install-node]
+
+// tag::prep[]
+
+== Preparing for Installation
 
 Install Couchbase Server first, if you haven't done so already.
 Refer to the xref:6.0@server:install:install-intro.adoc[installation guide] for server installation instructions.
 
-[#install-node]
+The Couchbase Node.js Client will run on any https://github.com/nodejs/Release[supported LTS version of Node.js] -- currently, 6.x, 8.x, and 10.x.
+On the Windows platform, it may be necessary to https://developercommunity.visualstudio.com/content/problem/249011/update-nodeexe-at-least-from-541-to-6x-version.html[update the version of Node.js that Visual Studio points to].
+On non-windows platforms you will also need to install https://github.com/nodejs/node-gyp[node-gyp^]. This can be installed with `npm`, and should also be available in your distribution's package manager:
```

Comment:
node-gyp is automatically included when installing npm (or its automatically fetched as soon as it is needed).  It is a core part of the actual Node.js ecosystem.  It's probably worth simply mentioning that the user may need to have the appropriate build tools for their platform (Visual Studio/gcc/etc...) in order to install the extension, and they can refer to the node-gyp documentation for details on what is required.

Reply:
Changes made